tools/devops/automation/templates/loc-translations.yml

```diff
@@ -86,6 +86,7 @@ steps:
     packageSourceAuth: patAuth
     patVariable: '$(OneLocBuild--PAT)'
     isAutoCompletePrSelected: false
+    isUseLfLineEndingsSelected: true
```

Comment:
I suppose it is not CR LF.. the loc team would have make it more clear if it said is isUseCarageReturnInLineEndings.. 🤦